Source/GSXibLoader.m

```diff
@@ -272,10 +272,19 @@ - (NSNibConnector*) nibConnector
   [result setDestination: [self destination]];
   [result setSource: [self source]];
   [result setLabel: [self label]];
-  
+
   return result;
 }
 
+- (NSString*) description
+{
+  NSMutableString *description = [[super description] mutableCopy];
```

Comment:
Why not `NSString stringWithFormat: @"%@ - label: %@, source: %@, destination: %@", ...`?

Reply:
That is the way Eggplant writes description methods. I have changed it to the way we normally do this.